app/components/dashboard/show/collections_list_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render Elements::HeaderComponent.new('h3', classes: ['mb-4'], value: label) %>
```

Comment:
Will `h3` lead to gaps in the heading levels (accessibility problem)?

Alternative is something like `<h2 class="h3">`.

Also, FYI -- you can just do `classes: 'mb-4'` and the class merger will sort it out.

Reply:
I noticed we do this a lot in H2.. `<h1 class="h1...` and `<h3 class="h4...`, I was thinking those _should_ align, but probably not, so I went with `level` as requested, and kept `variant` as optional to add to the class.